src/Symfony/Component/Cache/Adapter/ApcuAdapter.php

```diff
@@ -92,7 +92,11 @@ protected function doDelete(array $ids)
     protected function doSave(array $values, $lifetime)
     {
         try {
-            return array_keys(apcu_store($values, null, $lifetime));
+            if (false === $failures = apcu_store($values, null, $lifetime)) {
```

Comment:
Just curious, do you guys treat is a good-practice coding style? Most of linting tools warn you about assignments being done inside the conditional statements.